src/api/app/views/webui/project/_basic_info.html.haml

```diff
@@ -4,11 +4,13 @@
       = category_badge(category)
   - if project.url.present?
     = link_to(project.url, project.url, class: 'mb-3 d-block')
-  #description-text
+  #description-text.mb-4
     - if project.description.blank?
       %i No description set
     - else
       = render partial: 'webui/shared/collapsible_text', locals: { text: project.description }
+  - project.label_templates.each do |label_template|
```

Comment:
This makes no sense, there should be a new show page for these, which is only accessible to maintainers linked on Project#show. Nobody else really needs to see the labels for the project on the project page itself

Reply:
Yes, I know, I published the PR to get a feel on how it looks

Reply:
Ah, you mean this is not the place you wanted to show the labels?

Reply:
Those are templates, I do not see the point of listing templates *anywhere* on the Project#show page, all we need is a link to a label templates index for the given project from which we will be able to manage the templates

Reply:
I moved the listing under a project's tab, as we discussed.